tools/socgen/socmap_gen.py

```diff
@@ -94,6 +95,10 @@
 DDR_HADDR["ariane"] = 0x800
 DDR_HADDR["ibex"] = 0x800
 
+PBS_HADDR = dict()
+PBS_HADDR["leon3"] = 0x500
+PBS_HADDR["ariane"] = 0xA00
```

Comment:
does this mean the bitstreams for Ariane get loaded at 0xA0000000? how was that decided?

Reply:
In baremetal, yes. This address is decided randomly (on the lower addresses of the DDR just to be safe)

Reply:
How is the address done for Linux? or it that still TBD?

Reply:
In Linux, the partial bitstreams are moved into (a non-predefined) the kernel memory space by the driver. Then the mem address is associated with the index of the bistream by the runtime manger (a kernel module)